src/draw/lv_draw_buf.c

```diff
@@ -258,22 +250,23 @@ lv_result_t lv_draw_buf_init(lv_draw_buf_t * draw_buf, uint32_t w, uint32_t h, l
     header->flags = 0;
     header->magic = LV_IMAGE_HEADER_MAGIC;
 
-    draw_buf->data = lv_draw_buf_align(data, cf);
```

Comment:
why change this?

Reply:
lv_draw_buf_init is normally used in `lv_bin_decoder` to convert a const image array to draw_buf struct. 

Instead of align the data secretly, which could lead to global buffer out of range access and wrong draw result if data is not aligned already, I think we should add warning below instead and keep the data unchanged.

The log message could be ERROR, or we can add assert here. What do you think?

Reply:
@XuNeo 
I think it is enough to print the error log and let the bin decoder do the copy + alignment.

Reply:
I have changed it back. Current image post process won't check address alignement and I don't think I should.
If the C-array image data start address is not aligned, error is reported and user should adjust the image source.